back-end/src/routes/Router.ts

```diff
@@ -11,3 +11,4 @@ router.post('/register', MainController.register);
 router.post('/login', MainController.login);
 router.post('/confirm', passport.authenticate('jwt', { session: false }),
 	MainController.confirm);
+router.post('/closeContactFlag', RequestController.getCloseContactFlag);
```

Comment:
This doesn't work because this router uses https and needs the HCP certificate.

Create another router, e.g.:
```suggestion
export const httpRouter = express.Router();
httpRouter.post('/closeContactFlag', RequestController.getCloseContactFlag);
```

And in App.ts:
```typescript
import { router, httpRouter } from './routes/Router';
app.use('/http', httpRouter);
```

Reply:
Note: Issue #75 is responsible for upgrading the http router to https.